<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaReservaUView</class>
 <widget class="QWidget" name="NuevaReservaUView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 248, 248);

QPushButton::disable{
	background-color: #f8f8f8;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 0, 206);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 800 30pt &quot;Roboto&quot;;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>NUEVA RESERVA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #4A0064;
border: none;
color: #FFFFFF;
font: 800 10pt &quot;Roboto&quot;;
}

QPushButton:hover{
background-color: #F94E4E;
}

QPushButton:pressed{
background-color: #EE2525;
}</string>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cbxOrigen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxDestino">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona un origen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona un destino</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pasajeros</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtPasajeros">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuscar">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #9900CE;
color: #ffffff;
font: 800 8pt &quot;Roboto&quot;;
border:none;
transition: all .5s;
}

QPushButton:hover{
	background-color: #6B0C8C
}

QPushButton:pressed{
	background-color: #A30DD8;
}</string>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 12pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string>BUSCAR UN VUELO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSiguiente">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #9900CE;
color: #ffffff;
font: 800 10pt &quot;Roboto&quot;;
border:none;
transition: all .5s;
}

QPushButton:hover{
	background-color: #6B0C8C
}

QPushButton:pressed{
	background-color: #A30DD8;
}</string>
   </property>
   <property name="text">
    <string>SIGUIENTE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 12pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string>PASO 1/2</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="txtFecha">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A partir de</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblVuelos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>710</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
